docker-compose.yaml

```diff
@@ -0,0 +1,75 @@
+version: '3.7'
+
+services:
+  boltstream:
+    container_name: boltstream_django
+    build: .
+    image: 'boltstream'
+    env_file: '.env'
+    #command: ["gunicorn", "-c", "gunicorn.conf.py", "app.wsgi:application"]
+    command: 'python manage.py runserver 0.0.0.0:${PORT}'
+    restart: unless-stopped
+    volumes:
+      - ./boltstream:/home/boltstream/app/boltstream
+    ports:
+      - ${PORT}:${PORT}
+    depends_on:
+      - postgres
+      - redis
+
+  postgres:
+    container_name: boltstream_postgres
+    image: postgres
+    restart: unless-stopped
+    environment:
+      - POSTGRES_DB=boltstream
+      - POSTGRES_USER=boltstream
+      - POSTGRES_PASSWORD=boltstream
+
+  nginx:
+    container_name: boltstream_nginx
+    build:
+      context: .
+      dockerfile: Dockerfile-nginx
+    restart: unless-stopped
+    env_file: '.env'
+    ports:
+      - "8080:8080"
+    volumes:
+      - ./docker/nginx/offline.mp4:/etc/nginx/offline.mp4
+      - ./docker/nginx/default_preview.mp4:/etc/nginx/default_preview.mp4
+      - ./docker/nginx/rtmp-exec.py:/etc/nginx/rtmp-exec.py
```

Comment:
I think maybe this wasn't included in the commit (along with other mp4 files)

Reply:
I'm thinking these common files should not be part of ansible directory structure. They are part of the application and should be packaged within the dockerfile. Perhaps a directory called `common` at the top-level?

@benwilber Can Ansible reference/include files outside of its own directory structure?